kafka/src/main/java/org/axonframework/extensions/kafka/eventhandling/consumer/streamable/KafkaTrackingToken.java

```diff
@@ -111,95 +101,83 @@ public static boolean isEmpty(KafkaTrackingToken token) {
     @SuppressWarnings("ConstantConditions")
     public static KafkaTrackingToken from(TrackingToken trackingToken) {
         isTrue(trackingToken == null || trackingToken instanceof KafkaTrackingToken,
-                () -> "Incompatible token type provided.");
+               () -> "Incompatible token type provided.");
         return trackingToken == null ? KafkaTrackingToken.emptyToken() : (KafkaTrackingToken) trackingToken;
     }
 
     /**
-     * Retrieve the {@link Map} stored in this {@link TrackingToken} containing partition-offset pairs corresponding the
-     * actual offsets per partition a Consumer is fetching records from.
+     * Retrieve the {@link TopicPartition}/offset {@link Map} stored in this {@link TrackingToken}. This collection
+     * corresponds to the actual progress of a Consumer polling for records.
      *
-     * @return the {@link Map} stored in this {@link TrackingToken} containing partition-offset pairs corresponding the
-     * actual offsets per partition a Consumer is fetching records from
+     * @return the {@link TopicPartition}/offset {@link Map} stored in this {@link TrackingToken}
      */
-    @SuppressWarnings("WeakerAccess")
-    public Map<Integer, Long> partitionPositions() {
-        return partitionPositions;
+    public Map<TopicPartition, Long> tokenPartitionPositions() {
```

Comment:
`tokenPartitionPositions`, or `topicPartitionPositions`? Actually, I would just go for `positions`, as the map makes it obvious that a position is returned for each TopicPartition.

Reply:
Fair one. Changed to `positions` and all of it's occurrence have also been renamed.